src/contexts/network.tsx

```diff
@@ -0,0 +1,79 @@
+import { createContext } from 'preact';
+import { useContext, useState, useEffect, useCallback } from 'preact/hooks';
+import { useSwitchChain } from 'wagmi';
+
+import { useLocalStorage, LocalStorageKeys } from '../hooks/useLocalStorage';
+import { NetworkIconType } from '../hooks/useGetNetworkIcon';
+import { ASSETHUB_ID } from '../constants/constants';
+
+export enum Networks {
+  AssetHub = 'AssetHub',
+  Polygon = 'Polygon',
+}
+
+interface NetworkContextType {
+  polkadotSelectedNetworkId: string;
```

Comment:
Again let's somehow make clear that this is only relevant for WalletConnect. Maybe rename the variable instead of adding a comment.